crafting_pieces.xml

```diff
@@ -6804,9 +6804,9 @@
       <Material id="Iron5" count="2" />
     </Materials>
   </CraftingPiece>
-  <CraftingPiece id="crpg_horseman_javelin_blade_h0" name="{=XSabsFbx}Weighted Awl Pike Head" tier="3" piece_type="Blade" mesh="spear_blade_37" length="23.7" weight="0.4104" excluded_item_usage_features="swing">
-    <BladeData stack_amount="3" physics_material="wood_weapon" body_name="bo_spear_b" holster_mesh="throwing_spear_quiver_5" holster_body_name="bo_throwing_spear_quiver_5" holster_mesh_length="96.7">
-      <Thrust damage_type="Pierce" damage_factor="0.77" />
+  <CraftingPiece id="crpg_horseman_javelin_blade_h0" name="{=XSabsFbx}Weighted Awl Pike Head" tier="3" piece_type="Blade" mesh="spear_blade_37" length="23.7" weight="0.8" excluded_item_usage_features="swing">

```

Comment:
Why has weight been added to the base item? Did this really need a nerf? The weight added is coumpounded by the extra ammo in the stack . I do not see horseman javelin being so meta that it needs weight added

Reply:
I spoke to Diggles about this - Weight was increased as it's designed to be used on horseback and not by infantry on the ground who just want more ammo - This helps to prevent infantry using it on foot.